ci/build-windows.yml

```diff
@@ -1,9 +1,23 @@
 steps:
   - checkout: self
-    fetchDepth: 1 # speed up checkout from 30sec to 7sec
-  - powershell: '.\dev-env\windows\dadew-installer.ps1'
-    displayName: 'Install dev-env'
-    #- script: ./ci/configure-bazel.ps1
-    #  displayName: 'Configure Bazel'
-  - powershell: '.\build.ps1'
-    displayName: 'Bazel Build'
+    fetchDepth: 2 # speed up checkout from 30sec to 7sec
+
+  - powershell: |
```

Comment:
I made `install/sync/enable` part of the `build.ps1`: https://github.com/digital-asset/daml/blob/master/build.ps1#L4